uniffi_bindgen/src/scaffolding/templates/RustBuffer.rs

```diff
@@ -1,71 +1,25 @@
 // Everybody gets basic buffer support, since it's needed for passing complex types over the FFI.
 
-/// This helper allocates a new byte buffer owned by the Rust code, and returns it
-/// to the foreign-language code as a `RustBuffer` struct. Callers must eventually
-/// free the resulting buffer, either by explicitly calling the destructor defined below,
-/// or by passing ownership of the buffer back into Rust code.
-#[doc(hidden)]
+#[allow(clippy::missing_safety_doc)]
 #[no_mangle]
 pub extern "C" fn {{ ci.ffi_rustbuffer_alloc().name() }}(size: i32, call_status: &mut uniffi::RustCallStatus) -> uniffi::RustBuffer {
-    uniffi::call_with_output(call_status, || {
-        uniffi::RustBuffer::new_with_size(size.max(0) as usize)
-    })
+    uniffi::ffi::uniffi_rustbuffer_alloc(size, call_status)
 }
 
-/// This helper copies bytes owned by the foreign-language code into a new byte buffer owned
-/// by the Rust code, and returns it as a `RustBuffer` struct. Callers must eventually
-/// free the resulting buffer, either by explicitly calling the destructor defined below,
-/// or by passing ownership of the buffer back into Rust code.
-///
-/// # Safety
```

Comment:
Should we keep the `Safety` here (maybe change it a little to reflect the scope of the function) - I see we disable it from clippy (reasonably because it's already in the underlying function) - but just in case users get curious and want to audit the generated code, what do you think?

Reply:
That works! We might want to link to a GitHub permalink, the `docs.rs` or something so users don't have to go looking where `uniffi/src` is